discord-bot/Dockerfile

```diff
@@ -3,10 +3,10 @@
 FROM node:19
 WORKDIR /usr/src/app
 
-COPY package.json dist/yarn.lock ./
+COPY dist/package.json dist/yarn.lock ./
 RUN yarn install --frozen-lockfile --production
 
-# Will have to rewrite this if we bring in outside dependencies
-COPY . ./
+# then copy over typescript payload
+COPY dist ./
 
-CMD ["node", "lib/index.js"]
+CMD ["node", "discord-bot/lib/index.js"]
```

Comment:
To my mind, the sane way to do this is to say, "the directory structure of dist/ is exactly the same as the directory structure of manifold/". Anything else is confusing. So I would have made the discord bot code go in `dist/backend/discord-bot`, not `dist/discord-bot`.

Reply:
At the time, `discord-bot` wasn't in the `backend` directory, so this wasn't right, but now it is and this is appropriate advice!